usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_award.md

```diff
@@ -0,0 +1,180 @@
+## [Spending by Award](#spending-by-award)
+**Route:** `/api/v2/search/spending_by_award/`
+
+**Method:** `POST`
+
+This route takes award filters and fields, and returns the fields of the filtered awards.
+
+### Request
+field: Defines what award variables are returned.
```

Comment:
The parameter is called `fields` (plural).
Please indicate this parameter takes an array.

Reply:
Indicate that the array values must be from the Fields list provided below.

Reply:
Done